konfera/views.py

```diff
@@ -1,14 +1,15 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
 from konfera.models import Event, Sponsor
 
 
-def sponsor_list_view(request):
-    events = Event.objects.all()
-    sponsors = Sponsor.objects.all()
-    context = {'events': events,
+def sponsor_list_view(request, event_id):
+    sponsors = Sponsor.objects.select_related('event') \
+                    .filter(event__id=event_id).order_by('type')
+    event = Event.objects.get(pk=event_id)
```

Comment:
to have ids from DB in urls is bad practice. Lets introduce slug field in event model. And update urls to contain this file, and update view logic search by slug field.

This way we give user ability to define his own url per event which is nice.